Client/src/components/profile.tsx

```diff
@@ -0,0 +1,67 @@
+import '../css/profile.css';
+import { User} from "@auth0/auth0-react";
+import { useAppSelector } from '../features/hooks';
+import apiService from '../services/apiService';
+import { Buffer } from "buffer";
+import { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { refreshData } from '../features/user_slice';
+
+function Profile() {
+  const [username,setUsername] = useState('');
+  const user  = useAppSelector((state)=>state.user);
+  const [img,setImg] = useState()
+
+  useEffect(()=>{
+    const base64String:any = Buffer.from(user.avatar.data.data).toString("base64");
```

Comment:
This can be stored in the redux store as a base64 so no need to useEffect here. Change reducer action - NEW TASK

Reply:
Changed